examples/apexlog/main.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/apex/log"
 )
 
-var ProjectId int64 = 363389
-var ProjectKey string = "baa755018d5e35e07897ee0087fcce9c"
+var ProjectId int64 = 105138                               // Insert your Project Id here
+var ProjectKey string = "fd04e13d806a90f96614ad8e529b2822" // Insert your Project Key here
```

Comment:
Change many digits of the project key so that it's definitely not valid. Also, change the project id to something like `999999` so that it's more obvious that the user should change it.

Reply:
@ChimanJain before you merge can you replace this ProjectKey with something like you did in the other main.go file:

```
var ProjectId int64 = 999999                               // Insert your Project Id here
var ProjectKey string = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" // Insert your Project Key here
```